engine/src/test/java/com/google/android/fhir/search/SearchTest.kt

```diff
@@ -1761,7 +1761,7 @@ class SearchTest {
     val query =
       Search(ResourceType.Patient)
         .apply {
-          has<Condition>(Condition.SUBJECT) {
+          has(ResourceType.Condition, Condition.SUBJECT) {
```

Comment:
nit: can you add the argument names for better readability:

```suggestion
          has(resourceType = ResourceType.Condition, referenceParam = Condition.SUBJECT) {
```

Reply:
:+1: Done.